container/Dockerfile

```
@@ -261,7 +261,9 @@ ENV VIRTUAL_ENV=/opt/dynamo/venv \
 # Install common and test dependencies
 RUN --mount=type=bind,source=./container/deps/requirements.txt,target=/tmp/requirements.txt \
     --mount=type=bind,source=./container/deps/requirements.test.txt,target=/tmp/requirements.test.txt \
-    uv pip install --requirement /tmp/requirements.txt --requirement /tmp/requirements.test.txt
+    UV_GIT_LFS=1 uv pip install \
```

is git-lfs available inside container already? we dont need to apt install it?

---

You are right, the flag lets it silently fail. It is now installed with 20643f4